app/forms/RadioButtonAmountForm.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.validation.mappings.MappingUtil.optionCurrency
+import play.api.data.Forms.{of, tuple}
+import play.api.data.format.Formatter
+import play.api.data.{FieldMapping, Form, FormError}
+
+object RadioButtonAmountForm {
```

Comment:
Should this have its own unit tests?

Reply:
I was thinking to continue as we're going (as we dont' have unit tests for forms as they're tested in the IT test) and then apply the unit tests once we pick up the task to move the integration tests to unit test

Reply:
OK.